apps/api/src/planning-session/planning-session.controller.ts

```diff
@@ -18,9 +20,15 @@ import { PlanningSessionService } from './planning-session.service';
 @Controller('sessions')
 export class PlanningSessionController {
   constructor(private planningSessionService: PlanningSessionService) {}
+
```

Comment:
Shouldn't there be a guard for this controller?

Reply:
if you're wondering about the Auth Guard, that is available globally. Only authenticated users can access this controller.